src/graphorm/index.ts

```diff
@@ -171,17 +174,40 @@ const obj = new GraphORM({
           isMany: false,
           customRelation: (ctx, parentAlias, childAlias, qb): QueryBuilder =>
             qb
-              .select('*')
-              .from(SourceMember, 'sm')
-              .where(`sm."userId" = :userId`, { userId: ctx.userId })
-              .andWhere(`sm."sourceId" = "${parentAlias}".id`),
+              .addSelect('"userId"')
```

Comment:
Please don't add a select here, if `userId` is always needed add it as a required column in `SourceMember`

Reply:
Cool. Let me add it as required 🚢

Reply:
Worked like a magic! 🧙